### DISCUSSION TITLE
Confusion about DropdownMenu animation classes

### DISCUSSION THREAD

**Discussion by kjrocker** [shadcn-ui/ui#3512] [General]
I'm working with the Dropdown Menu, and it's not animating out of the box.

I dug into it, and the classes like `fade-in-0` don't seem to exist in Tailwind?

Same for `zoom-out-95`, `slide-in-from-top-2`, basically the whole animation section is made up of classes I can't find documentation for.

I'm sure I'm missing something. Is there a tailwind plugin I need to add to get all these fancy classes?

**Comment by ezeparziale** [comment]
Check https://www.npmjs.com/package/tailwindcss-animate

**Reply by kjrocker** [reply]
Yep, that has all the missing classes, and works perfectly.

Maybe I should file this as a documentation issue, I can't find anything about ShadeCN requiring `tailwind-animate`

**Reply by devsmartproject** [reply]
### Ask
When you install shadcn-ui, tailwindcss-animate is automatically installed, why didn't it work for you?  🧐

**Reply by HammadAwan424** [reply]
I also encountered this issue. I was using js therefore coundn't install shadcn-ui without ts and didn't got tailwind-animate package. Thanks.
